src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,36 @@
-import React from 'react';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import { Post } from '../../types/Post';
+
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo = ({ post }: Props) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        {post.user && <UserInfo user={post.user} />}
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    {
```

Comment:
This is how prettier formatted for me this piece of code, matter of preference but this is more common way of implementing an if else in jsx:
```tsx
{post.comments.length === 0 ? (
  <b data-cy="NoCommentsMessage">No comments yet</b>
) : (
  <CommentList comments={post.comments} />
)}
```